.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1056,29 +1064,6 @@ stages:
           testResultsFiles: tracer/build_data/results/**/*.trx
         condition: succeededOrFailed()
 
-    - job: native
-      timeoutInMinutes: 60 #default value
-      steps:
-      - template: steps/clone-repo.yml
-        parameters:
-          targetShaId: $(targetShaId)
-          targetBranch: $(targetBranch)
-      - template: steps/install-dotnet.yml
-      - template: steps/restore-working-directory.yml
-
-      - script: tracer\build.cmd BuildAndRunNativeUnitTests
-        displayName: Build and Test
-        env:
-          DD_LOGGER_DD_API_KEY: $(ddApiKey)
-
-      - task: PublishTestResults@2
-        displayName: publish test results
-        inputs:
-          testResultsFiles: |
-            tracer/test/**/test*.xml
-            profiler/build_data/*.xml
-        condition: succeededOrFailed()
```

Comment:
We will need to publish the test results from the build stage now I think

Reply:
It's done here https://github.com/DataDog/dd-trace-dotnet/pull/5614/files#diff-2e09bd52f55cbb3965ceeae7f556dbbd458b94a4566d9631bea1204a4ad12198R385

Reply:
ah ok, I thought that was just logs :+1: